crates/pallets/starknet/src/lib.rs

```diff
@@ -409,11 +409,9 @@ pub mod pallet {
     /// EVENTS
     /// See: `<https://docs.substrate.io/main-docs/build/events-errors/>`
     #[pallet::event]
```

Comment:
Can you remove the whole Event section? If it is not used at all

Reply:
done